src/metabase/public_settings.clj

```diff
@@ -507,6 +506,17 @@
                  (validate-help-url new-value-string)
                  (setting/set-value-of-type! :string :help-link-custom-destination new-value-string))))
 
+
+(defsetting show-metabase-links
+  (deferred-tru
+   (str
+    "Whether or not to display Metabase links outside admin settings."))
```

Comment:
nit - we usually keep Clojure lines to 120 chars, so there's nothing wrong with having this on one line
```suggestion
  (deferred-tru (str "Whether or not to display Metabase links outside admin settings."))
```